src/status.rs

```diff
@@ -28,7 +28,7 @@ use std::cmp::Ordering;
 /// Registry](http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml) which is
 /// the source for this enum (with one exception, 418 I'm a teapot, which is
 /// inexplicably not in the register).
-#[derive(Debug, Hash)]
+#[derive(Debug, Hash, Copy, Clone)]
```

Comment:
The manual implementations were deliberate actually, because it's faster to compile than rustc generating the huge match statements for this big enum...